### PR TITLE
Universal method/config importer/exporter

### PR DESCRIPTION
**Pull request by MattPutnam**
Completely overhauled the existing method-config-importer to be a single component that can import/export methods and configs from the workspace->method configs tab dialog or the Method Repository.

Note: this functionality is dependent on the corresponding Orchestration PR.

TODOs include:
* Cleaning up some code duplication (DSDEEPB-1361)
* Displaying more information about the config/method to be imported
* Better error messages in the case of a conflict

### CODE REVIEW

**Comment by eddiebroad** [comment]
Looks great in the UI!

**Comment by eddiebroad** [comment]
By "Displaying more information about the config/method to be imported" you mean things such as the inputs, outputs, the method it came from?  Would such information go to the right of where it says "Save As"?  Are the TODOs planned for *this* PR? or for another/later PR(s)?

**Comment by MattPutnam** [comment]
Displaying more info: There's a section at the top of the second page with name, namespace, and ID.  It might also be nice to show synopsis, description, maybe even an option to expand and show payload, but right now I don't have any meaningful data to look at.

File: src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs [hunk 1]
```diff
@@ -83,3 +83,9 @@
                              :WebkitTouchCallout "none" :WebkitUserSelect "none"
                              :KhtmlUserSelect "none" :MsUserSelect "none"}} props)
    children])
+
+(defn create-link [onClick & children]
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs]: Good call. Can you add a story to our backlog to update other call sites?

**Comment by dmohs** [comment]
:+1: Good to merge after rebase.

**Pull request merged**